CMakeLists.txt

```diff
@@ -40,7 +40,11 @@ set_property(   GLOBAL PROPERTY PROPERTY_OUTPUT_FORMAT elf32-little${ARCH})
 # compiler options needed by all source files. All zephyr libraries,
```

Comment:
There is something wrong with the commit message.

It cuts off mid-sentence.

Reply:
well, you wrote that comment yourself in commit 12f7f761, so I can't guess what your meaning were.
I only appended the text with linker flags appended afterwards.

But isn't the text just a new line and the sentence continues. 
At least that's how I read it:
All zephyr libraries, including the library named "zephyr" link with this library to obtain these flags.

Reply:
Commit message, not comment.

Reply:
Ahh, ok.

I think the reason might be the reference to issue number was #8440 was moved to new line, due to 72 char limit.
But # at line start comments out the line :-s